.eslintrc.json

```diff
@@ -1,7 +1,8 @@
 {
   "extends": [
     "react-app",
-    "prettier"
+    "prettier",
+    "stylelint"
```

Comment:
This wont be run on scss files as long as you do not change the "lint" script in the package.json

Reply:
I removed it from the current PR. I could not get it working right away so I created #168. That way we will have a single PR for the formatting of all the styles.